### ISSUE TITLE
ERNIE 2.0: A Continual Pre-training Framework for Language Understanding

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/phd-bib#38]
## [ERNIE 2.0: A Continual Pre-training Framework for Language Understanding](https://arxiv.org/abs/1907.12412)
*Recently, pre-trained models have achieved state-of-the-art results in various language understanding tasks, which indicates that pre-training on large-scale corpora may play a crucial role in natural language processing. Current pre-training procedures usually focus on training the model with several simple tasks to grasp the co-occurrence of words or sentences. However, besides co-occurring, there exists other valuable lexical, syntactic and semantic information in training corpora, such as named entity, semantic closeness and discourse relations. In order to extract to the fullest extent, the lexical, syntactic and semantic information from training corpora, we propose a continual pre-training framework named ERNIE 2.0 which builds and learns incrementally pre-training tasks through constant multi-task learning. Experimental results demonstrate that ERNIE 2.0 outperforms BERT and XLNet on 16 tasks including English tasks on GLUE benchmarks and several common tasks in Chinese. The source codes and pre-trained models have been released at this https URL.*

### Cite this paper
```bibtex
@article{sun2019ernie,
  title={{Ernie 2.0: A continual pre-training framework for language understanding}},
  author={Sun, Yu and Wang, Shuohuan and Li, Yukun and Feng, Shikun and Tian, Hao and Wu, Hua and Wang, Haifeng},
  journal={arXiv preprint arXiv:1907.12412},
  year={2019}
}
```

**Comment by juanmc2005** [comment]
### Summary

The paper introduces an improvement over the previously introduced ERNIE 1.0 model (from the same authors), describing a framework for continual multi-task learning.

Apart from the fact that they get better results than other transformer models (although I don't know to which point the comparison is fair), they propose interesting pretraining tasks to replace or augment the classic MLM and NSP objectives. However, I find disappointing in the way they handled CL, as they train on all previous tasks at each "step" or "trajectory", only limiting the number of iterations per task for efficiency. As a result, the model does not learn to accumulate knowledge, but rather it is almost retrained at each step with a warm start.

Regarding the comparison with other SOTA models, I wouldn't be able to say to which extent such a framework constitutes an advantage over (let's say) BERT, as they do not train on the same data. It could be possible that their data is just way better than a wikipedia dump and their framework doesn't give any advantage over a standard BERT, so I'm not quite convinced by the results.
